src/components/Authentication/Login.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import { useAuth } from "../../contexts/AuthDialogContext";
+import { useModal } from "../../contexts/ModalContext";
```

Comment:
suggestion: use an index file in `contexts` and export all the files of that folder from there to get cleaner imports